README.md

```diff
@@ -599,6 +599,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Cahllagerfeld](https://github.com/Cahllagerfeld)
   - [Carlie Hope](https://github.com/carliemaria)
   - [Carmen Chow](https://github.com/cchow33)
+  - [Carlos Mayofret](https://github.com/carlosmf996)
```

Comment:
Your name should be below Carlie Hope, in line 601.
Please update accordingly. If you need any help, let us know. 🙂